test/cypress/integration/27-expenses.test.js

```diff
@@ -263,6 +263,7 @@ describe('New expense flow', () => {
         cy.get('input[name="description"]').type('Service Invoice');
         cy.get('input[name="items[0].amount"]').type('{selectall}4200');
 
+        cy.wait(10000);
```

Comment:
Waiting 10 seconds means making the test 10 seconds slower forever.

Is there an alternative?

Reply:
@znarf : Ah, yes. This is more of a temporary change. With the upgrades to Cypress, I think the test runner has got faster and better; but the problem with that is we then hit our rate limiter (https://github.com/opencollective/opencollective-api/blob/4a90fb584acad19c2ea8b8007688817822ec884c/server/graphql/v2/mutation/ExpenseMutations.ts#L306-L309). Note that the rate limiters time is 10 seconds. This is why I set it to 10 seconds so that it's guaranteed to not hit. The root cause of this is that our rate limiter here is set to skip on `NODE_ENV="test"`; but on our CI we use `NODE_ENV="ci"` (https://github.com/opencollective/opencollective-frontend/blob/9c9e73555a755b36f89bb142079fb1d4646fb547/.github/workflows/e2e.yml#L13). We need to change this NODE_ENV variable to `test` in our CI environments I think. I need to investigate more on this as I don't know exactly why we have set that value to `ci`. 😄

Reply:
We should make the Rate Limiter configurable and don't do rate limiting in CI. Unless this is the exact want to test.

We should never have non-configurable Rate Limiters.

Reply:
> We should make the Rate Limiter configurable and don't do rate limiting in CI. Unless this is the exact want to test.
> 
> We should never have non-configurable Rate Limiters.

@znarf : Just to confirm, when you say configurable, you mean adding another env variable so that the rate limiter can be enabled or disabled? 🤔

Reply:
@SudharakaP yes, activated/disabled and the rate itself.

Reply:
@znarf : So as per our discussion I've fixed this problem at https://github.com/opencollective/opencollective-api/pull/5989. 😄